Mathlib/Algebra/Order/Field/Basic.lean

```diff
@@ -162,21 +163,24 @@ lemma mul_le_of_nonneg_of_le_div (hb : 0 ≤ b) (hc : 0 ≤ c) (h : a ≤ b / c)
   · rwa [le_div_iff hc] at h
 #align mul_le_of_nonneg_of_le_div mul_le_of_nonneg_of_le_div
 
+@[bound]
 theorem div_le_one_of_le (h : a ≤ b) (hb : 0 ≤ b) : a / b ≤ 1 :=
   div_le_of_nonneg_of_le_mul hb zero_le_one <| by rwa [one_mul]
 #align div_le_one_of_le div_le_one_of_le
 
+@[bound]
 lemma mul_inv_le_one_of_le (h : a ≤ b) (hb : 0 ≤ b) : a * b⁻¹ ≤ 1 := by
   simpa only [← div_eq_mul_inv] using div_le_one_of_le h hb
 
+@[bound]
 lemma inv_mul_le_one_of_le (h : a ≤ b) (hb : 0 ≤ b) : b⁻¹ * a ≤ 1 := by
   simpa only [← div_eq_inv_mul] using div_le_one_of_le h hb
 
 /-!
 ### Bi-implications of inequalities using inversions
 -/
 
-@[gcongr]
+@[gcongr, bound]
```

Comment:
Are there cases where `gcongr` is appropriate but `bound` is not? Either way, can you answer this in the docstring of `@[bound]`?

Reply:
Done.